lisa/features/startstop.py

```diff
@@ -48,3 +48,6 @@ def restart(self, wait: bool = True) -> None:
         self._log.info("restarting")
         self._restart(wait=wait)
         self._node.close()
+
+    def get_status(self) -> str:
```

Comment:
It needs to define the return value as an enum in the common class, and map it to str in the Azure class.